packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -85,28 +87,24 @@ export class ConfigUtil {
     return undefined;
   }
 
-  public static async getUsername(
-    projectPath: string
-  ): Promise<string | undefined> {
-    const configAggregator = await ConfigUtil.getConfigAggregator(projectPath);
+  /**
+   * Get the username of the currently auth'd user for the project.
+   *
+   * @returns The username for the configured Org if it exists.
+   */
+  public static async getUsername(): Promise<string | undefined> {
```

Comment:
With the change to use workspace context util, this function is no longer used right now. Although it looks like it might be useful to add to the configUtil that is currently in core (once we move it here into utils).  Will leave it up to you to either remove or leave it.

Reply:
word. Yeah will leave it for now as I think we'll make use of it shortly.